scripts/utils.js

```diff
@@ -29,3 +29,13 @@ export function getLibraryDependencyVersion(libraryName) {
 
   return `${dependency.majorVersion}.${dependency.minorVersion}`;
 }
+
+/**
+ * @param {string} translationKey 
```

Comment:
I think every JSDoc comment for a function should contain one line that explains what the function does, and every parameter and the return value should at least briefly explain what they are used for/what to expect. That's not relevant if one thinks about potentially checking type-safety only (if one just sees JSDoc for that purpose), but the comments/descriptions are actually what I think are useful when you try to understand the code.

I for my part would for instance like to know that `rest` means "Placeholders and values to replace in the text." (https://github.com/h5p/h5p-editor-php-library/blob/2f319eb31a16644bb5e864413926d88f8873954b/scripts/h5peditor-editor.js#L610) ;-)

Reply:
Thanks for the thorough review! The JSDoc was meant for typing only, and as this is no more than a wrapper function to move the library name out of the code, I didn't see the point of documenting it further. I do agree with you on typing `rest` in a better way, and renaming it.